doc/source/cluster/kubernetes/examples/mnist-training-example.md

```diff
@@ -0,0 +1,122 @@
+(kuberay-mnist-training-example)=
```

Comment:
Can you install [Vale](https://docs.ray.io/en/master/ray-contribute/docs.html#how-to-use-vale) and ensure there are no Vale warnings?